src/index.js

```diff
@@ -5,6 +5,7 @@ var main = function(options) {
   console.log("Gathering Manifest data...");
   var resources = WalkManifest(options.decrypt, options.output, options.input);
 
+  console.log("Downloading additional data...");
```

Comment:
It always bothered me that we didn't log something here and just started to spew out downloads.